lib/src/text_view.dart

```diff
@@ -90,6 +95,7 @@ class _RichTextViewState extends State<RichTextView> {
               TextSpan(text: ' \u2026'),
               TextSpan(
                   text: _expanded ? widget.viewLessText : widget.viewMoreText,
+                  style: widget.viewMoreLessStyle,
```

Comment:
Hi @iabdousd  sorry for the very late response. 
In that case, it's better to add it on line 106 instead as `style: widget.viewMoreLessStyle ?? linkStyle`